### ISSUE TITLE
Reloadium fails to start

### ISSUE THREAD

**Issue by hyabean** [reloadware/reloadium#178]
## Describe the bug*

 'RW_IDE_NAME': 'PyCharm 2023.3.2',
 'RW_IDE_PLUGINVERSION': '1.3.4',

```Traceback (most recent call last):
  File "C:\Users\Administrator\.reloadium\package\3.9\reloadium\corium\l111ll1111ll1l11Il1l1.py", line 189, in ll1l1ll11l1lllllIl1l1
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll111111llll1l1lIl1l1.py", line 69, in exec_module
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 559, in l1l1ll1ll111111lIl1l1
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 605, in ll1111l11l1l1ll1Il1l1
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 116, in lllll1l111111ll1Il1l1
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\l1lll11111lll111Il1l1\ll111l1l1llll11lIl1l1.py", line 80, in visit
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 122, in visit_Module
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\l1lll11111lll111Il1l1\ll111l1l1llll11lIl1l1.py", line 88, in generic_visit
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\l1lll11111lll111Il1l1\ll111l1l1llll11lIl1l1.py", line 80, in visit
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 156, in visit_FunctionDef
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 129, in visit_ClassDef
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\l1lll11111lll111Il1l1\ll111l1l1llll11lIl1l1.py", line 88, in generic_visit
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\l1lll11111lll111Il1l1\ll111l1l1llll11lIl1l1.py", line 80, in visit
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 149, in visit_FunctionDef
  File "D:\a\reloadware\reloadware\reload\package\__obfuscated__\reloadium\fast\ll1lll11l1l11l1lIl1l1\llllllll11l1l11lIl1l1.py", line 106, in ll1ll1ll111lll11Il1l1
  File "<string>", line 9, in __init__
  File "C:\Users\Administrator\.reloadium\package\3.9\reloadium\corium\ll1lll11l1l11l1lIl1l1\l1ll1l11111111llIl1l1.py", line 513, in __post_init__
  File "C:\Users\Administrator\.reloadium\package\3.9\reloadium\corium\ll1lll11l1l11l1lIl1l1\l1ll1l11111111llIl1l1.py", line 375, in __post_init__
  File "C:\Users\Administrator\.reloadium\package\3.9\reloadium\corium\ll1lll11l1l11l1lIl1l1\l1ll1l11111111llIl1l1.py", line 428, in l1l1llll1l111l1lIl1l1
  File "C:\ProgramData\anaconda3\envs\xbrl\lib\tokenize.py", line 512, in _tokenize
    raise IndentationError(
  File "<tokenize>", line 4
    _parser = XMLParser(recover=True, huge_tree=True, target=checkFileType())
IndentationError: unindent does not match any outer indentation level
(4.5398) Critical reloader Error
{'traceback': 'Traceback (most recent call last):\n'
              "  File '.../reloadium\\corium\\l111ll1111ll1l11Il1l1.py', line "
              '189, in ll1l1ll11l1lllllIl1l1\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll111111llll1l1lIl1l1.py', "
              'line 69, in exec_module\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 559, in l1l1ll1ll111111lIl1l1\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 605, in ll1111l11l1l1ll1Il1l1\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 116, in lllll1l111111ll1Il1l1\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\l1lll11111lll111Il1l1\\ll111l1l1llll11lIl1l1.py', "
              'line 80, in visit\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 122, in visit_Module\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\l1lll11111lll111Il1l1\\ll111l1l1llll11lIl1l1.py', "
              'line 88, in generic_visit\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\l1lll11111lll111Il1l1\\ll111l1l1llll11lIl1l1.py', "
              'line 80, in visit\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 156, in visit_FunctionDef\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 129, in visit_ClassDef\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\l1lll11111lll111Il1l1\\ll111l1l1llll11lIl1l1.py', "
              'line 88, in generic_visit\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\l1lll11111lll111Il1l1\\ll111l1l1llll11lIl1l1.py', "
              'line 80, in visit\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 149, in visit_FunctionDef\n'
              '  File '
              "'D:\\a\\reloadware\\reloadware\\reload\\package\\__obfuscated__\\reloadium\\fast\\ll1lll11l1l11l1lIl1l1\\llllllll11l1l11lIl1l1.py', "
              'line 106, in ll1ll1ll111lll11Il1l1\n'
              "  File '<string>', line 9, in __init__\n"
              '  File '
              "'.../reloadium\\corium\\ll1lll11l1l11l1lIl1l1\\l1ll1l11111111llIl1l1.py', "
              'line 513, in __post_init__\n'
              '  File '
              "'.../reloadium\\corium\\ll1lll11l1l11l1lIl1l1\\l1ll1l11111111llIl1l1.py', "
              'line 375, in __post_init__\n'
              '  File '
              "'.../reloadium\\corium\\ll1lll11l1l11l1lIl1l1\\l1ll1l11111111llIl1l1.py', "
              'line 428, in l1l1llll1l111l1lIl1l1\n'
              '  File '
              "'C:\\ProgramData\\anaconda3\\envs\\xbrl\\lib\\tokenize.py', "
              'line 512, in _tokenize\n'
              '    raise IndentationError(\n'
              "  File '<tokenize>', line 4\n"
              '    _parser = XMLParser(recover=True, huge_tree=True, '
              'target=checkFileType())\n'
              'IndentationError: unindent does not match any outer indentation '
              'level\n'}
Reloadium experienced a fatal error and has to quit.
To see the exception run Reloadium with environmental variable RW_DEBUG=True
Please submit a github issue to let us know at https://github.com/reloadware/reloadium```

**Comment by hyabean** [comment]
sample code

```import sys

def my_func():
    class checkFileType(object):
        def close(self):pass
    _parser = 'test'
    print(_parser)

if __name__ == '__main__':
    my_func()```

**Comment by dkrystki** [comment]
Fixed in Reloadium 1.4.0 and PyCharm plugin 1.4.0
